src/domain/Configuration.ts

```diff
@@ -20,6 +20,7 @@ interface Configuration {
     tableName: string;
     // TODO: Only 'sql' is supported sourceType now. JS will be supported later.
     sourceType: 'sql' | 'javascript' | 'typescript';
+    stubPath?: string;
```

Comment:
Updating the yml file every time we create a migration might be a bit inconvenient. Should it be a cli option?

Reply:
The inconvenience is mainly because we expect changes in sync-db.yml to be committed. But in the case of migrations, we might need multiple create or alter migrations in one go. Which doesn't really work well with a yml file that is tracked in git or other version control tool.